pkg/asset/images.go

```diff
@@ -2,10 +2,12 @@ package asset
 
 // DefaultImages are the defualt images bootkube components use.
 var DefaultImages = ImageVersions{
-	Busybox:         "busybox",
 	Etcd:            "quay.io/coreos/etcd:v3.1.8",
 	EtcdOperator:    "quay.io/coreos/etcd-operator:v0.3.2",
 	Flannel:         "quay.io/coreos/flannel:v0.7.1-amd64",
+	FlannelCNI:      "quay.io/coreos/flannel-cni:0.1.0",
```

Comment:
Where can I find the source of the `flannel-cni` image? I need some newer binaries* and just stumbled on this.

\* for the portmap plugon to work (https://github.com/containernetworking/plugins/issues/43)